lib/devices/ios/xcode.js

```diff
@@ -82,7 +84,11 @@ xcode.getVersion = function (cb) {
     }
     // we want to read the CFBundleShortVersionString from Xcode's plist.
     // It should be in /[root]/XCode.app/Contents/
-    var plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+    try {
+      var plistPath = escapeSpace(xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist"));
```

Comment:
i don't think we actually want to escape the path here. fs.existsSync doesn't have a problem with spaces in the path, and in fact it will return the wrong thing if the spaces are escaped. we just want to escape the plistPath in the `cmd` down below.